### ISSUE TITLE
Conditional javascript block

### ISSUE THREAD

**Issue by berniegp** [thymeleaf/thymeleaf#486]
Hello,

I cannot find a way to add a javascript block inside an `th:inline="javascript"` `<script>` tag. I read the docs and tried many things to no avail. Here is what I want to do:

```
<script type="text/javascript" th:inline="javascript">
jQuery(function($) {

    // other js code which must run BEFORE the "admin" block
    var options = ...

    <th:block th:if="${isAdmin}"> // <-- can't do this
        // This MUST NOT be present on the rendered page unless the user
        // is an admin. This is just one example. There are other cases
        // where some JS code must be conditionally added at a specific
        // location in a <script> tag.
        options.annoyingClientFlag = [[ ${isAnnoyingClient} ]];
    </th:block>

    // other js code which must run AFTER the "admin" block
    ui.init(options);
});
</script>
```

It seems that this feature, if not already present, would be very useful.

Relevant SO question: http://stackoverflow.com/questions/31771821/thymeleaf-how-to-add-conditional-block-in-javascript

**Comment by berniegp** [comment]
Implemented (I think) by #395

**Comment by danielfernandez** [comment]
Exactly. I'm afraid there is no way to do what you need in Thymeleaf 2.1, but the new textual template modes (and inlining) in Thymeleaf 3.0 will allow you to do it using the new syntax.
